trunk/SUAVE/Methods/Noise/Fidelity_One/Propeller/propeller_noise_frequency.py

```diff
@@ -0,0 +1,327 @@
+# propeller_noise_frequency.py
+# 
+# Created:  May 2016, Carlos
+# Modified: 
+
+# ----------------------------------------------------------------------        
+#   Imports
+# ----------------------------------------------------------------------    
+
+import numpy as np
+import scipy
+import matplotlib.pyplot as plt
+
+from scipy import special
+from SUAVE.Core import Units, Data
+
+
+def propeller_noise_frequency(noise_data):
+    
```

Comment:
Needs a docstring

Reply:
I removed this file from this PR and will put it in a noise PR